packit_service/worker/helpers/testing_farm.py

```diff
@@ -1458,6 +1458,23 @@ def _payload_installability(self, distro: str, compose: str) -> dict:
             },
         }
 
+    @implements_fedora_ci_test("rpminspect")
+    def _payload_rpminspect(self, distro: str, compose: str) -> dict:
+        git_repo = "https://github.com/fedora-ci/rpminspect-pipeline.git"
+        git_ref = (
+            commands.run_command(["git", "ls-remote", git_repo, "HEAD"], output=True)
+            .stdout.strip()
+            .split()[0]
+        )
```

Comment:
IIRC you can do

```suggestion
        git_ref = "master"
```

Since you resolve `HEAD` anyways, I don't really see any benefit to this… On the contrary, if you proposed adding a tag to the config to improve the reproducibility, I would understand :D